packages/opds-web-client/src/components/Collection.tsx

```diff
@@ -93,8 +93,6 @@ export default class Collection extends React.Component<CollectionProps, {}> {
                 url={this.props.collection.url}
                 lanes={this.props.collection.lanes}
                 updateBook={this.props.updateBook}
-                fulfillBook={this.props.fulfillBook}
-                indirectFulfillBook={this.props.indirectFulfillBook}
```

Comment:
@adriana-alter same issue as before! This time it is:
```
Root.tsx -> Collection.tsx -> Lanes.tsx -> Lane.tsx -> Book.tsx -> DownloadButton.tsx
```

A long way to send these two actions for DownloadButton to use.